mk/uvmt/README.md

```diff
@@ -15,20 +15,33 @@ To run the UVM environment you will need:
 - the RISC-V GCC compiler, and
 - at least some familiarity with [Make](https://www.gnu.org/software/make/manual/) and Makefiles.
 
-COREV Environment Variables
+Required COREV Environment Variables
+------------------------------------
+The following environment variables **must** be set for any `make` invocation to run tests or set in the user's environment to customize setting.
+
+| Environment Variable | Description  |
+|----------------------|--------------|
+| CV_SIMULATOR         | The default simulator to use for all tools (dsim, vcs, xrun, vsim, riviera).  Can be overridden on any make invocation. |
+| CV_CORE              | The core to simulate by default.  Can be overridden on any make command line.  Typically specified in the same case as the directory in which the testbench resides (e.g. cv32e40p, cva6).  However makefiles have access to internally defined CV_CORE_LC and CV_CORE_UC variables to get expected cases of the CV_CORE name (e.g. for macros, etc that might typically be capitalized). |
+| CV_SW_TOOLCHAIN      | Points to SW toolchain installation for compiling, assembling, and/or linking test programs. |
+| CV_SW_PREFIX         | Prefix of the SW toolchain installation. |
```

Comment:
This replaces `CV_SW_VENDOR`, right?  (With slightly changed meaning/definition)

Reply:
Agreed - I prefer the CV_SW_TOOLCHAIN to be the full path to the binary - as long as the /lib, /inc etc. directories does not need to be specified? (do we need RISCV to point to the top level directory of the toolchain?)

Reply:
If there is no code that uses the TOOLCHAIN or PREFIX part in isolation then it is fine.

I did a quick search and found that `uvmt.mk` does `PATH=$(RISCV)/bin:$(PATH)` (where "RISCV" comes from "..._TOOLCHAIN"). But I this line should not be necessary.

It is not an easy thing to grep for, so I don't know if this is the whole picture.

Reply:
PREFIX is used all over the place, so I have decided to hold my nose and maintain the status quo with both `CV_SW_TOOLCHAIN` and `CV_SW_PREFIX` used to generate the full path to the tools.

One day, I will live to regret this decision.